### PR TITLE
Improved documentation code readability by reducing the number of imports and adding spaces around some variables.

### PR DESCRIPTION
**Pull request by ZeroCool940711**

### CODE REVIEW

**Review by ndonkoHenri — CHANGES_REQUESTED** [review]
I noticed that some areas do not respect the PEP8 coding style guidelines.
Read the 5th point here: https://peps.python.org/pep-0008/#other-recommendations

A simple/little example of what i mean, in your code is the `ft.app(target = main)`
PEP8 suggests using: `ft.app(target=main)`
_Notice the space around the `=`._

If I have some spare time I might try correcting the 'shot'.

**Comment by ndonkoHenri** [comment]
Sorry, I use the 'review changes', instead of just commenting.
I made no changes actually.

**Comment by FeodorFitsner** [comment]
Yep, this is what took my attention as well: spaces around `=` in keyword arguments :)

I think we should follow PEP8. Intuitively, it "feels" nicer when there are spaces around, but kwargs without spaces is commonly accepted thing. Found this SO discussion: https://stackoverflow.com/questions/8853063/pep-8-why-no-spaces-around-in-keyword-argument-or-a-default-parameter-value

Anyway, all Python files in Flet should be formatted with Black: https://github.com/flet-dev/flet/blob/main/CONTRIBUTING.md#code-formatting

**Comment by ZeroCool940711** [comment]
> I noticed that some areas do not respect the PEP8 coding style guidelines. Read the 5th point here: https://peps.python.org/pep-0008/#other-recommendations
> 
> A simple/little example of what i mean, in your code is the `ft.app(target = main)` PEP8 suggests using: `ft.app(target=main)` _Notice the space around the `=`._

Yeah, I wasn't sure about `target = main` and a few others, I was expecting this kind of answer, I tried to use PEP8 as much as possible but I was doing most of the changes either manually or using find and replace all so, a few lines going through without me noticing and not following PEP8 was inevitable.

I will try to fix this and push the changes to this PR. Feel free to point out other places where I should change things to follow PEP8.

**Comment by ZeroCool940711** [comment]
> Yep, this is what took my attention as well: spaces around `=` in keyword arguments :)
> 
> I think we should follow PEP8. Intuitively, it "feels" nicer when there are spaces around, but kwargs without spaces is commonly accepted thing. Found this SO discussion: https://stackoverflow.com/questions/8853063/pep-8-why-no-spaces-around-in-keyword-argument-or-a-default-parameter-value
> 
> Anyway, all Python files in Flet should be formatted with Black: https://github.com/flet-dev/flet/blob/main/CONTRIBUTING.md#code-formatting

For Python code I have enabled on both VS Code and WingIDE black auto-formatting but because of how I was doing the changes to the docs using both VS Code to replace some stuff and also doing manual edits on the Markdown files it was hard to make sure things were formatted properly. Sorry about that.

**Review by ndonkoHenri — COMMENTED** [review]
I think it looks better(pep8-speaking) now. 🙂

**Comment by ndonkoHenri** [comment]
Sure. But users must be aware that it is possible to use OOP in Flet too.

**Comment by ZeroCool940711** [comment]
> Sure. But users must be aware that it is possible to use OOP in Flet too.

As I said, it should be optional but still be there for advanced users or for when we get used to stuff on flet and you learn more about OOP, I personally suck with OOP as I had never a need for it so, even thought I've being coding with python for 10 years I barely know some OOP, I have learned more about it when trying to learn things for flet this last days than what I knew about it before, still, having the option to start crawling and then walk or run would be nice.

**Comment by ndonkoHenri** [comment]
You are right. That is pretty advanced for the newcomers. 
Let's let the final words to @FeodorFitsner (Flet's BDFL).

**Comment by ZeroCool940711** [comment]
Just resolved some conflicts on this PR from recently changes on the user control page, hope it can be merged soon, let me know if there is anything else I need to fix or improve on it.

File: blog/2022-08-04-gradients-button-textfield-styles.md
Comment (FeodorFitsner) [inline blog/2022-08-04-gradients-button-textfield-styles.md:252]: There are still inconsistencies like this `= `across multiple samples. I understand it's a lot, but could could you fix all kwargs to don't have spaces around `=`. It's easier to spot such places before merging a PR. Thank you!

Comment (ZeroCool940711) [reply blog/2022-08-04-gradients-button-textfield-styles.md:252]: Ok, I will fix them, I will go over the whole docs again and fix those issues.

Comment (ZeroCool940711) [reply blog/2022-08-04-gradients-button-textfield-styles.md]: I think things are good now, do you mind giving it a second look?

**Comment by ndonkoHenri** [comment]
https://peps.python.org/pep-0008/#other-recommendations
The second point and third points.

Here are some areas where the white space on both sides of the '=' (the assignment operator) where removed. 

(I am using a GitHub mobile, hence can't help change that.)

image

image

**Comment by FeodorFitsner** [comment]
So, ready to merge? :)

**Comment by ZeroCool940711** [comment]
@FeodorFitsner It looks good to me, I went and double-checked everything and it looks good to me, if you think it's good then let's merge it.

**Comment by FeodorFitsner** [comment]
Sounds great, merging!

**Pull request merged**
